docusaurus/docs-guides/hsm/yubikey.mdx

```diff
@@ -1,1124 +1,1114 @@
-are easily searched for using your search engine of choice.
-
-:::caution
-This quickstart intended audience is for more technically savvy indiviuals. You will need to be familar with the command
-line interface of your operating system.
-:::
-
-## Prerequistites
-
-* [YubiKey 5 nfc](https://www.yubico.com/product/YubiKey-5-nfc) - clearly you'll need one in order to use this
-  quickstart!
-* [OpenSC](https://github.com/OpenSC/OpenSC/wiki) is installed and `pkcs11-tool` is either on the PATH or at a known
-  location. Not required however this quickstart uses the `pkcs11-tool` to illustrate that the device is PKCS#11
-  compliant. HSM manufacturers will generally provide a similar tool and often expand it's usage. See more below.
-  [yubico-piv-tool](https://developers.yubico.com/yubico-piv-tool/) - YubiKey privides a similar tool to the
-  `pkcs11-tool`. This tool is needed to be installed because it contains the pkcs#11 module (driver) for the HSM. As
-  this is a tool specific to Yubico we've chosen to not use this in the following commands.
-* Ensure the YubiKey is factory reset. To avoid any compliations with existing information in the YubiKey ensure the
-  device is factory reset using the [YubiKey
-  Manager](https://www.yubico.com/products/services-software/download/YubiKey-manager/).
-* In order to successfully use the YubiKey the libraries provided by the `yubico-piv-tool` *MUST* either be on the path
-  or in a common location that is known to the OS. On linux this is likely done by the YubiKey software installation but
-  on Windows you'll need to take any additional actions highlighted in the Windows-specific sections.
-* Linux Only: If you're using linux - you'll need to follow the [build
-  instructions](https://developers.yubico.com/yubico-piv-tool/) provided by Yubico. Before you can do anything with the
-  Yubikey you'll need to make sure the `libykcs11.so` exists on your system. When creating this quickstart the library
-  was built to `./yubico-piv-tool-2.0.0/ykcs11/.libs/libykcs11.so`.
-* [Ubuntu](https://ubuntu.com/) was used to test this guide. An attempt was also made with [linux
-  mint](https://linuxmint.com/) however the attempt failed when trying to compile the Yubikey software and was aborted.
-  It would likely have worked if enough effort was put into discovering why that linux variant had issues pulling and
-  compiling the necessary software. If you see strange errors when following this guide and are not using Ubuntu it may
-  be related.
-* ziti and ziti-tunnel are both downloaded and on the path.
-
-## Let's Use the YubiKey!
-
-Here's the list of steps we'll accomplish in this quickstart:
-
-* Establish a bunch of environment variables to make it easy to copy/paste the other commands. You'll want to *look at
-  these environment variables*. They need to be setup properly. If you have problems with this guide it is almost

```

Comment:
why are you removing these words?

Reply:
This hunk should've been in the other PR. Not pertinent to broken links.